packages/blade/build.config.ts

```diff
@@ -0,0 +1,62 @@
+import { resolve } from 'node:path';
+import { defineBuildConfig } from 'unbuild';
+
+export default defineBuildConfig({
+  alias: {
+    '@/private': resolve(__dirname, 'private'),
+    '@/public': resolve(__dirname, 'public'),
+
+    // Deno requires Node.js primitives to be imported with the `node:` prefix
+    // and certain dependencies that Blade uses don't do that, so we're aliasing
+    // them ourselves.
+    'fs/promises': 'node:fs/promises',
+    crypto: 'node:crypto',
+    events: 'node:events',
+    fs: 'node:fs',
+    http: 'node:http',
+    http2: 'node:http2',
+    os: 'node:os',
+    path: 'node:path',
+    stream: 'node:stream',
+  },
+  entries: [
```

Comment:
I believe unbuild might not support globs like tsup does, but it seems like it supports specifying directories instead of file paths, which might capture all files inside of those directories, like we had it before?

https://github.com/unjs/unbuild/blob/8924db79846916962d488193dbe6a1553f490af7/examples/2.mkdist/build.config.ts#L6-L10